src/wasm/literal.cpp

```diff
@@ -33,6 +33,67 @@ Literal::Literal(const uint8_t init[16]) : type(Type::v128) {
   memcpy(&v128, init, 16);
 }
 
+Literal::Literal(const Literal& other) {
+  type = other.type;
+  switch (type.getSingle()) {
+    case Type::i32:
+    case Type::f32:
+      i32 = other.i32;
+      return;
+    case Type::i64:
+    case Type::f64:
+      i64 = other.i64;
+      return;
+    case Type::v128:
+      memcpy(&v128, other.v128, 16);
+      return;
+    case Type::funcref:
+      func = other.func;
+      return;
+    case Type::exnref:
+      new (&exn) auto(std::make_unique<ExceptionPackage>(*other.exn));
```

Comment:
This seems complicated. Does `exn = std::make_unique<ExceptionPackage>(*other.exn)` not work?

Reply:
I mindlessly copied this from the implementation of the copy constructor, in which I tried to use placement new.. but anyway now we are calling `operator=` and this isn't even necessary. Done.

Reply:
No we can't do that, because now we are using this code in the copy constructor too. If we do `exn = ...` this will trigger the deletion of the current `exn`, which will in turn trigger deletion of its contained `ExceptionPackage`, which does not yet exist.